README.md

```diff
@@ -50,6 +50,15 @@ $> composer require palantirnet/the-vagrant
 
 You can re-run the install script later if you need to change your configuration.
 
+## Upgrading
+
+To upgrade the-vagrant in a project, you will need to:
+
+1. `composer update palantirnet/the-vagrant`
+2. Follow any steps from the [release notes](https://github.com/palantirnet/the-vagrant/releases).
+
+*Note:* If you need to update your VM, such as [drupalbox](https://app.vagrantup.com/palantir/boxes/drupalbox), you will need to run `vagrant destroy` then `vagrant box update` and `vagrant up`. Updating the VM doesn't always require updating The Vagrant.
```

Comment:
@byrond this is great! It might also be helpful to mention that it is necessary any time the release notes don't explicitly say that it is not required.

Reply:
Great idea! I'll have to double-check with @becw to make sure that is true for releases that don't say that explicitly. Alternatively, we should probably just be explicit either way in the release notes. Thanks for the feedback!